src/cli/print.go

```diff
@@ -52,6 +51,6 @@ func PrintError(messages ...interface{}) {
 func PrintLabelAndValue(label, value string) {
 	labelFmt := color.New(color.Bold).Add(color.Underline)
 	PrintlnColor(labelFmt, label+":")
-	Println(util.Indent(value))
+	Println(Indent(value))
```

Comment:
This command is so specific to printing output indented that it belongs to the `cli` package instead of a shared `util` folder.